src/LiveComponent/assets/src/Component/plugins/LoadingPlugin.ts

```diff
@@ -215,21 +195,18 @@ export default class implements PluginInterface {
 }
 
 const parseLoadingAction = function(action: string, isLoading: boolean) {
-    switch (action) {
-        case 'show':
-            return isLoading ? 'show' : 'hide';
-        case 'hide':
-            return isLoading ? 'hide' : 'show';
-        case 'addClass':
-            return isLoading ? 'addClass' : 'removeClass';
-        case 'removeClass':
-            return isLoading ? 'removeClass' : 'addClass';
-        case 'addAttribute':
-            return isLoading ? 'addAttribute' : 'removeAttribute';
-        case 'removeAttribute':
-            return isLoading ? 'removeAttribute' : 'addAttribute';
+    const actions = new Map(Object.entries({
+        'show': isLoading ? 'show' : 'hide',
+        'hide': isLoading ? 'hide' : 'show',
+        'addClass': isLoading ? 'addClass' : 'removeClass',
+        'removeClass': isLoading ? 'removeClass' : 'addClass',
+        'addAttribute': isLoading ? 'addAttribute' : 'removeAttribute',
+        'removeAttribute': isLoading ? 'removeAttribute' : 'addAttribute',
+    }));
+    if (!actions.has(action)) {
```

Comment:
Why create a Map just to fetch from it one time? The switch case seems at least as good